src/GeoBounds.h

```diff
@@ -1,21 +1,30 @@
-// Copyright © Matt Jones and Contributors. Licensed under the MIT License (MIT). See LICENCE.md in the repository root for more information.
+// Copyright © Matt Jones and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
```

Comment:
Your licence header is becoming America 😄

Reply:
OOF. How did that happen? I'll fix that.